cis_cell.py

```diff
@@ -0,0 +1,34 @@
+# from cis_dna import
+import cis_config as conf
+import protocol_pb2 as proto
+import uuid
+
+
+def move(cell):
+    pass
+
+
+def is_alive(cell):
+    starvation_threshold = conf.ENERGY_THRESHOLD
+    if cell.energy_level < starvation_threshold:
+        return False
+    else:
+        return True
+
+
+def divide(cell):
```

Comment:
`divide` either returns a new cell or `None`, can we change the name to reflect that? Sth. along `maybe_divide` (also not great but you get the idea)?

Reply:
Do you think it would more sense to outsource the check for the energy level into `cell_computing_service.py`?

Reply:
I have no strong preference here